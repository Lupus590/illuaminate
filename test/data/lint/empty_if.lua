-- config: (lint (only (syntax:empty-if)))
if x then
elseif y then
elseif z then
  print()
else
end
