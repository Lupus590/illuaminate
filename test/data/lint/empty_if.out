empty_if.lua:[2:0-2:9]: Empty if clause [syntax:empty-if]
   │
 2 │ if x then
   │ ^^^^^^^^^
empty_if.lua:[3:0-3:13]: Empty elseif clause [syntax:empty-if]
   │
 3 │ elseif y then
   │ ^^^^^^^^^^^^^
empty_if.lua:[6:0-6:4]: Empty else clause [syntax:empty-if]
   │
 6 │ else
   │ ^^^^
No errors and 3 warnings
@@ -3,6 +3,5 @@
 elseif y then
 elseif z then
   print()
-else
 end
 
